src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Extensions/ApprenticeshipFundingDtoExtensionsTests.cs

```diff
@@ -94,11 +94,11 @@ public void CalcFundingForDate_Calculates_Correctly_For_2_Apprentices_24_Months(
             {
                 Duration = 24,
                 MaxEmployerLevyCap = 20_000,
-                EffectiveFrom = new DateTime(DateTime.UtcNow.Year, DateTime.UtcNow.AddMonths(-1).Month, 1),
+                EffectiveFrom = new DateTime(2021, DateTime.UtcNow.AddMonths(-1).Month, 1),
                 EffectiveTo = null
             };
 
-            var result = sut.CalcFundingForDate(2, new DateTime(DateTime.UtcNow.Year, 10, 1));
+            var result = sut.CalcFundingForDate(2, new DateTime(2021, 10, 1));
 
             Assert.AreEqual(8_000, result);
         }
```

Comment:
On all this file, pls remove the DateTime.UtcNow and use hard coded values.

Reply:
@SHESFA I have done these changes as part of another PR.

Reply:
@jawwadbaig That's fine, we just make sure the PR with good changes goes to master _after_ the other PRs, otherwise above changes will overwrite any _good_ changes.  Or best to do them here as well.  Just the tests you changed (2 of them?).

Reply:
@SHESFA yes good idea. I will make those changes here as well. Then there will be no conflicts.

Reply:
These tests are all up to date now.